### PR TITLE
Casminst 3137

### PR DESCRIPTION
**Pull request by cdelatte-hpe**

### CODE REVIEW

**Review by kimjensen-hpe — APPROVED** [review]

File: install/deploy_management_nodes.md [hunk 1]
```diff
@@ -507,7 +519,7 @@ The configuration workflow described here is intended to help understand the exp
     The field `available` would be `True` if Ceph sees the drive as empty and can
     be used, e.g.:
 
-    ```
+    ```bash
```
Comment (brantk-hpe) [inline install/deploy_management_nodes.md:522]: this doesn't contain a bash script.

File: install/deploy_management_nodes.md [hunk 2]
```diff
@@ -525,9 +537,12 @@ The configuration workflow described here is intended to help understand the exp
     ncn-s# cephadm shell -- ceph-volume inventory --format json-pretty | jq -r '.[]|select(.available==true)|.path'
     ```
 
-1. Wipe the drive ONLY after you have confirmed the drive is not being used by the current Ceph cluster via options 1, 2, or both.
+##### Wipe and Add Drives
+
+1. Wipe the drive `ONLY after you have confirmed the drive is not being used by the current Ceph cluster` via options 1, 2, or both.
```
Comment (brantk-hpe) [inline install/deploy_management_nodes.md:542]: Is the goal to emphasize the text? Should be using bold or italics not monospace.

**Review by brantk-hpe — APPROVED** [review]

File: install/deploy_management_nodes.md [hunk 3]
```diff
@@ -426,6 +436,8 @@ The configuration workflow described here is intended to help understand the exp
     24
     ```
 
+   **`IMPORTANT:`** If the returned number of OSDs is == total_osds calulated, then you can skip the following steps.  If not, then please proceed with the below additional checks and remdiation steps.
```
Comment (bo-quan) [inline install/deploy_management_nodes.md:439]:
```suggestion
   **`IMPORTANT:`** If the returned number of OSDs is equal to total_osds calculated, then you can skip the following steps.  If not, then please proceed with the below additional checks and remediation steps.
```

File: install/deploy_management_nodes.md [hunk 4]
```diff
@@ -470,7 +482,7 @@ The configuration workflow described here is intended to help understand the exp
     24
     ```
 
-    If the numbers are equal, then you may need to fail your `ceph-mgr` daemon to get a fresh inventory.
+    If the numbers are equal, but less than the `total_osds` calulated, then you may need to fail your `ceph-mgr` daemon to get a fresh inventory.
```
Comment (bo-quan) [inline install/deploy_management_nodes.md:485]:
```suggestion
    If the numbers are equal, but less than the `total_osds` calculated, then you may need to fail your `ceph-mgr` daemon to get a fresh inventory.
```

**Review by bo-quan — CHANGES_REQUESTED** [review]
Some typos and minor suggestion

**Review by bo-quan — APPROVED** [review]

**Review by denniswalker — APPROVED** [review]

**Pull request merged**